README.md

```diff
@@ -28,6 +28,7 @@ After you're finished please remove all the comments and instructions!
 
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
+    - [Key Features](#key-features)
   - [💻 Getting Started ](#-getting-started-)
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
```

Comment:
- [ ] You missed adding column `species` in the animals' table in the `schema.sql` file. Please make sure to add that as requested in the project requirements.